src/v/cluster/tests/shard_placement_table_test.cc

```diff
@@ -238,7 +238,7 @@ class reconciliation_backend
     }
 
     ss::future<result<ss::stop_iteration>>
-    reconcile_ntp_step(const model::ntp& ntp, ntp_reconciliation_state& rs) {
+    reconcile_ntp_step(const model::ntp& ntp, ntp_reconciliation_state&) {
```

Comment:
so hilarious. its like clang17 could catch 900 of these and clang18 can now catch 902